docs/set-up-your-environment.md

```diff
@@ -0,0 +1,104 @@
+---
+id: set-up-your-environment
+title: Set Up Your Environment
+hide_table_of_contents: true
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import constants from '@site/core/TabsConstants';
+
+import GuideLinuxAndroid from './\_getting-started-linux-android.md';
+import GuideMacOSAndroid from './\_getting-started-macos-android.md';
+import GuideWindowsAndroid from './\_getting-started-windows-android.md';
+import GuideMacOSIOS from './\_getting-started-macos-ios.md';
+
+In this guide, you'll learn how to set up your environment, so that you can run your project with Android Studio and Xcode. This will allow you to develop with Android emulators and iOS simulators, build your app locally, and more.
+
+:::note
+This guide requires Android Studio or Xcode. If you already have one of these programs installed, you should be able to get up and running within a few minutes. If they are not installed, you should expect to spend about an hour installing and configuring them.'
+
+<details>
+<summary>Is setting up my environment required?</summary>
+
+Setting up your environment is not required if you're using a Framework. With that, if you prefer to develop with Android Studio and/or Xcode, you can set up your local computer while still using a Framework.
+
+If you have constraints that prevent you from using a Framework, or you'd like to write your own Framework, then setting up your local environment is a requirement. After your environment is set up, learn how to [get started without a framework](get-started-without-a-framework).
+
+</details>
+:::
+
+#### Development OS
+
+<Tabs groupId="os" queryString defaultValue={constants.defaultOs} values={constants.oses} className="pill-tabs">
+<TabItem value="macos">
+
+#### Target OS
+
+<Tabs groupId="platform" queryString defaultValue={constants.defaultPlatform} values={constants.platforms} className="pill-tabs">
+<TabItem value="android">
+
+[//]: # 'macOS, Android'
+
+<GuideMacOSAndroid/>
+
+</TabItem>
+<TabItem value="ios">
+
+[//]: # 'macOS, iOS'
+
+<GuideMacOSIOS/>
+
+</TabItem>
+</Tabs>
+
+</TabItem>
+<TabItem value="windows">
+
+#### Target OS
+
+<Tabs groupId="platform" queryString defaultValue={constants.defaultPlatform} values={constants.platforms} className="pill-tabs">
+<TabItem value="android">
+
+[//]: # 'Windows, Android'
+
+<GuideWindowsAndroid/>
+
+</TabItem>
+<TabItem value="ios">
+
+[//]: # 'Windows, iOS'
+
+## Unsupported
+
+> A Mac is required to build projects with native code for iOS. You can follow the **Expo Go Quickstart** to learn how to build your app using Expo instead.
```

Comment:
Missing link?

Reply:
Also here you introduce "Expo Go" in this page without explaining what it is

Reply:
Good catch! This is in the current docs and I forgot to update this instance.